docs/api.md

```diff
@@ -589,3 +589,252 @@ state.
 
 Please note only the existence of the annotation is important to treat the BMH
 as detached and the value of the annotation is always ignored.
+
+## HostFirmwareSettings
+
+A **HostFirmwareSettings** resource is used to manage BIOS settings for a host,
+there is a one-to-one mapping with **BareMetalHosts**.  A
```

Comment:
Bikeshedding: here "mapping" sounds a little bit strange, I was expecting something like  "one-to-one relationship"

Reply:
OK, I think in this context 'mapping' and 'relationship' are basically the same, but changed to use relationship.